packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -47,9 +47,10 @@ export class PostPublishPanel extends Component {
 		// Automatically collapse the publish sidebar when a post
 		// is published and the user makes an edit.
 		if (
-			prevProps.isPublished &&
-			! this.props.isSaving &&
-			this.props.isDirty
+			( prevProps.isPublished &&
+				! this.props.isSaving &&
+				this.props.isDirty ) ||
+			! this.props.postlink
```

Comment:
Thank you! Let's use `getCurrentPostId` selector though instead of `getEditedPostAttribute( 'id' )` and name the prop `currentPostId` for clarity.

Reply:
@ntsekouras I have implemented the changes!
Also, can you highlight on why using `getCurrentPostId ` is a better approach?

Thanks 🙌🏻

Reply:
Sure. Since we only want the `id`, `getCurrentPostId` returns that directly from the state whereas `getEditedPostAttribute` is a doing more and we don't need to fetch all the edits etc..

Reply:
@ntsekouras okay understood!
Is this PR now ready to be approved or does this need anymore changes?